lib/cylc/remote.py

```diff
@@ -168,8 +168,9 @@ def construct_ssh_cmd(raw_cmd, user=None, host=None, forward_x11=False,
         user_at_host += 'localhost'
     command.append(user_at_host)
 
-    # Pass cylc version (and optionally UTC mode) through.
-    command += ['env', quote(r'CYLC_VERSION=%s' % CYLC_VERSION)]
+    # Pass CYLC_VERSION, CYLC_CONF_PATH and optionally CYLC_UTC through.
+    command += ['env', quote(r'CYLC_VERSION=%s' % CYLC_VERSION),
+                quote('CYLC_CONF_PATH=%s' % os.environ.get('CYLC_CONF_PATH'))]
```

Comment:
Note: this is necessary for auto-restarted suites to pick up the test configuration (that one took a wile to debug!)